osmosis-1/osmosis-1.assetlist.json

```diff
@@ -1155,6 +1155,39 @@
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/fet.svg"
       },
       "coingecko_id": "fetch-ai"
+    },
+    {
+      "description": "The native token of Carbon",
+      "denom_units": [
+        {
+          "denom": "ibc/8FEFAE6AECF6E2A255585617F781F35A8D5709A545A804482A261C0C9548A9D3",
+          "exponent": 0,
+          "aliases": [
+            "swth"
```

Comment:
```
"aliases": [
            "dswth"
```
 If  `dswth` is unit of SWTH, so 1 SWTH = 100 000 000 dswth

Reply:
`swth` is the unit denom on the source chain (Carbon). We are creating a new alias here so that the UI can show the 8 d.p. which is used for human.

If we swap this, the denoms won't match the source chain - do you think that is okay?